build.gradle.kts

```diff
@@ -25,7 +25,7 @@ repositories {
 dependencyManagement {
   imports {
     mavenBom("org.springframework.cloud:spring-cloud-dependencies:2023.0.0")
-    mavenBom("io.awspring.cloud:spring-cloud-aws-dependencies:3.1.1")
+    mavenBom("io.awspring.cloud:spring-cloud-aws-dependencies:3.1.0")
```

Comment:
Try switching these around if you're having dependency version issues

Reply:
Switching the code around with dependency version `spring-cloud-aws-dependencies:3.1.1` doesn't seem to work on my local. It keeps giving error like `Error polling for message in queue. software.amazon.awssdk.services.sqs.model.SqsException: Service returned HTTP status code 400 (Service: Sqs, Status Code: 400, Request ID: null)` with this change when other setting keeps unchanged.
There may be other dependencies on my side not compatible with this version?